src/io/jesi/backpack/string.cljc

```diff
@@ -15,10 +15,8 @@
   ([prefix separator s]
    (let [match (str prefix separator)]
      (if (string/starts-with? s match)
-       (subs s (dec (count match)))
-       (if (= separator ", ")
-         (remove-prefix prefix \space s)
-         s)))))
+       (subs s (dec (+ 1 (count match))))
```

Comment:
Had to add 1 onto the count so it would add spaces at the start of the string

Reply:
`dec` stands for decrement which means subtract 1
```clojure
(= dec (partial - 1))